src/components/Form/FormFormats.js

```diff
@@ -1354,60 +1357,22 @@ export default {
-                        label: 'Day of Week',
-                        required: 'true',
-                        ...selectField(DAY_OF_WEEK_OPTIONS),
-                    },
-                    {
-                        name: 'startTime1',
-                        label: 'Start Time',
-                        required: 'true',
-                        component: <Fields.TimePicker format='hh:mm a' />,
-                        validator: Yup.date(),
-                    },
-                    {
-                        name: 'endTime1',
-                        label: 'End Time',
-                        required: 'true',
-                        component: (
-                            <Fields.TimePicker width={50} format='hh:mm a' />
-                        ),
-                        validator: Yup.date(),
-                    },
-                    {
-                        name: 'weekday2',
-                        label: 'Day of Week',
-                        ...selectField(DAY_OF_WEEK_OPTIONS),
-                    },
-                    {
-                        name: 'startTime2',
-                        label: 'Start Time',
-                        component: <Fields.TimePicker format='hh:mm a' />,
-                        validator: Yup.date(),
-                    },
-                    {
-                        name: 'endTime2',
-                        label: 'End Time',
-                        component: <Fields.TimePicker format='hh:mm a' />,
-                        validator: Yup.date(),
-                    },
-                    {
-                        name: 'weekday3',
-                        label: 'Day of Week',
-                        ...selectField(DAY_OF_WEEK_OPTIONS),
+                        name: 'CourseAvailability1',
+                        label: 'CourseAvailability1',
+                        component: <CourseAvailabilityField count={1} />,
+                        validator: Yup.mixed(),
                     },
                     {
-                        name: 'startTime3',
-                        label: 'Start Time',
-                        component: <Fields.TimePicker format='hh:mm a' />,
-                        validator: Yup.date(),
+                        name: 'CourseAvailability2',
+                        label: 'CourseAvailability2',
+                        component: <CourseAvailabilityField count={2} />,
+                        validator: Yup.mixed(),
                     },
                     {
-                        name: 'endTime3',
-                        label: 'End Time',
-                        component: <Fields.TimePicker format='hh:mm a' />,
-                        validator: Yup.date(),
+                        name: 'CourseAvailability3',
+                        label: 'CourseAvailability3',
+                        component: <CourseAvailabilityField count={3} />,
+                        validator: Yup.mixed(),
```

Comment:
Will we need to handle an arbitrary amount of course availability settings in the future?

Reply:
potentially, but I'm ok hard coding this for now. We can open this for a future issue if the need comes up